src/app/containers/Include/index.jsx

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable react/no-danger */
+import React, { useEffect, useState } from 'react';
+import 'isomorphic-fetch';
+import { string } from 'prop-types';
+import { GridItemConstrainedMedium } from '#lib/styledGrid';
+import webLogger from '#lib/logger.web';
+import useToggle from '../Toggle/useToggle';
+
+const logger = webLogger();
+
+const IncludeContainer = ({ href }) => {
+  const [markup, setMarkup] = useState(null);
+  const { enabled } = useToggle('styIncludes');
+
+  useEffect(() => {
+    const fetchMarkup = async () => {
+      const res = await fetch(href);
+      res
+        .text()
+        .then(html => setMarkup(html))
+        .catch(e => logger.error(`HTTP Error: "${e}"`));
+    };
+    fetchMarkup();
+  }, [href]);
+
+  if (markup && enabled) {
```

Comment:
We should probably check `enabled` before the return function. At the moment, this component will still make a request even if Includes are disabled.

Reply:
@simonsinclair we can do the check inside the hook before running the `fetchMarkup` because of eslint flagging when we put conditionals before hooks